virtual_rainforest/models/animals/scaling_functions.py

```diff
@@ -9,22 +9,8 @@
 
 from math import ceil
 
-from virtual_rainforest.models.animals.constants import (
-    DAMUTHS_LAW_TERMS,
-    ENDOTHERM_METABOLIC_RATE_TERMS,
-    ENERGY_DENSITY,
-    FAT_MASS_TERMS,
-    INTAKE_RATE_TERMS,
-    MUSCLE_MASS_TERMS,
-)
-
-
-def damuths_law(
-    mass: float,
-    taxa: str,
-    diet: str,
-    terms: dict = DAMUTHS_LAW_TERMS,
-) -> int:
+
+def damuths_law(mass: float, terms: tuple) -> int:
```

Comment:
This has only just occurred to me, but most of the functions in here are allometric laws. I can see the value in documenting each function, but the name for each law could be described in the docs and then apply a generic allometric function within the animal cohort calculations. This would reduce the number of functions and testing quite a lot.

As a sketch, you could do this. Note something else that just occurred to me: you can use `*argvalue` in a python function call to expand an iterable of values into positional arguments, which would get rid of all of the `terms[0]` and `terms[1]`.

```python
from math import ceil

def allometric_law(mass, coef, exp):

    return coef * mass ** exp

mass = 0.23
terms = (0.75, 0.047)

allometric_law(100, *terms)  # unpacks to allometric_law(100, 0.75, 0.047)
ceil(allometric_law(100, *terms))
```

Again I don't think this is necessary for this PR, but I think it is probably cleaner.

Reply:
I think I got `exp` and `coef` the wrong way around in that sketch 😄

Reply:
Alrighty, I've done most of this but will leave the totally-bespoke and allometric_laws portions for now.